lib/CMakeLists.txt

```diff
@@ -728,7 +728,7 @@ if(APPLE)
 elseif (NOT ANDROID)
   vs_lib_link_libraries("PUBLIC_COMMON" Threads::Threads)
 endif(APPLE)
-if(TEGRA_DEMO OR WINDOWS)
+if(WINDOWS)
```

Comment:
This change seems out of place, why'd you remove the TEGRA_DEMO?

Reply:
Indeed, I slipped a little bit here